mappings/net/minecraft/client/render/LayeredVertexOutput.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/class_4597 net/minecraft/client/render/LayeredVertexOutput
+	CLASS class_4598 Immediate
+		FIELD field_20952 fallbackBuilder Lnet/minecraft/class_287;
+		FIELD field_20953 layerBuilders Ljava/util/Map;
+		FIELD field_20954 currentLayer Ljava/util/Optional;
+		FIELD field_20955 startedBuilders Ljava/util/Set;
+		METHOD <init> (Lnet/minecraft/class_287;Ljava/util/Map;)V
+			ARG 1 fallbackBuilder
+			ARG 2 layerBuilders
+		METHOD method_22993 draw ()V
+		METHOD method_22994 draw (Lnet/minecraft/class_1921;)V
+			ARG 1 layer
+		METHOD method_22995 getConsumer (Lnet/minecraft/class_1921;)Lnet/minecraft/class_287;
+			ARG 1 layer
+	METHOD getBuffer getVertexConsumer (Lnet/minecraft/class_1921;)Lnet/minecraft/class_4588;
```

Comment:
Uh no

Reply:
Oops, renamed it at its usage and didn't realize it was a Mojang name. I'll fix this.

Reply:
Well, if mojang did not use this as a method reference at where they render the breaking progress, we could have named it with much more ease instead